How-To-Release-Riak.md

```diff
@@ -0,0 +1,56 @@
+This doc contains the internal Basho policy for building, testing, and releasing Riak.
+
+# Scheduling
+All public releases must conform to the following schedule:
+* An expected delivery date on a Wednesday. Each delivery date must have at least one fallback date that is no earlier than one week after the expected delivery date.
+* A Go/No-Go scheduled on the Thursday after the expected delivery date or fallback date to ensure that the public release can proceed as planned. During the Go/No-Go we must ensure:
+  * Giddyup has been triaged, and there are no major issues
+  * Smoke testing has completed successfully on all supported platforms
+  * Documentation content has been merged and triaged if necessary, and release notes are ready to ship
+  * Packages for all supported platforms have been built and are ready to stage
+  * Any marketing announcements have been reviewed and are ready to ship
+* A final staging date on the Friday after the Go/No-Go. During this time, all packages will be pushed to their final download locations, the `External Release` docs will be sent around internall for review, and the indexes for the download pages will be generated
+* A public release date on the Tuesday after the final staging date. This date cannot overlap with another public release.
+
+# Pre-Release
+All releases of Riak will have at least one `Release Candidate` before the final package build. These RCs will contain all of the `code, backports, and bugfixes` required to build the Final Release, but will not have any updated `Release Notes`. Release Notes updates will be the only difference between RCs and the final version that will be released to the public.
+
+## Pre-release Procedure
+All Release Candidates will undergo the following release stages to ensure quality:
+  * Package Build
```

Comment:
I was thinking that it would be evident in the doc I'm gonna drop in EE as well, but it probably makes more sense to call out each product in both places so there's no change of forgetting